storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/Changes.java

```diff
@@ -19,13 +21,37 @@
     private final Set<String> affectedTables;
 
     /**
+     * Immutable set of affected tags.
+     */
+    @NonNull
+    private final Set<String> affectedTags;
```

Comment:
+1 for `Object`. In my understanding, you can use as tag whatever you want and if `String` will be the most frequently used it doesn't mean that tag should be restricted to it. Also, `View` tag in Android is `Object` as well.